eth/backend.go

```diff
@@ -1154,44 +1153,6 @@ func (s *Ethereum) Etherbase() (eb libcommon.Address, err error) {
 	return libcommon.Address{}, errors.New("etherbase must be explicitly specified")
 }
 
-// isLocalBlock checks whether the specified block is mined
-// by local miner accounts.
-//
-// We regard two types of accounts as local miner account: etherbase
-// and accounts specified via `txpool.locals` flag.
-func (s *Ethereum) isLocalBlock(block *types.Block) bool { //nolint
```

Comment:
`isLocalBlock` and below `shouldPreserve` alongside its `Locals` config and cmd flags are all dead code that's never used